js/analyzer.js

```diff
@@ -194,29 +191,13 @@ YoastSEO.Analyzer.prototype.keyphraseSizeCheck = function() {
  * @returns resultObject
  */
 YoastSEO.Analyzer.prototype.keywordDensity = function() {
-	var result = [ { test: "keywordDensity", result: 0 } ];
-	if ( this.preProcessor.__store.wordcount > 100 ) {
-		var keywordDensity = this.keywordDensityCheck();
-		result[ 0 ].result = keywordDensity.toFixed( 1 );
-		return result;
-	}
-};
+	var getKeywordDensity = require( "./analyses/getKeywordDensity.js" );
+	var density = getKeywordDensity( this.preProcessor.__store.originalText, this.config.keyword );
+	var result = [ { test: "keywordDensity", result: density } ];
```

Comment:
In this PR we should still be checking here if the word count is higher then 100.